code/datums/trading/ai.dm

```diff
@@ -27,24 +27,66 @@ They sell generic supplies and ask for generic supplies.
 	)
 	possible_wanted_items = list(
 		/obj/item/device/                        = TRADER_SUBTYPES_ONLY,
+		/obj/item/device/kit					 = TRADER_BLACKLIST_ALL, // They're impossible to make or get outside of mining, just annoys traders.
 		/obj/item/device/assembly                = TRADER_BLACKLIST_ALL,
 		/obj/item/device/assembly_holder         = TRADER_BLACKLIST_ALL,
-		/obj/item/device/encryptionkey/syndicate = TRADER_BLACKLIST,
+		/obj/item/device/encryptionkey			 = TRADER_BLACKLIST_ALL, //Why does ai want NT encryption keys?
 		/obj/item/device/radio                   = TRADER_BLACKLIST_ALL,
 		/obj/item/device/pda                     = TRADER_BLACKLIST_SUB,
-		/obj/item/device/uplink                  = TRADER_BLACKLIST
+		/obj/item/device/chameleon				 = TRADER_BLACKLIST, // Why should it want a chameleon projector
+		/obj/item/device/dociler				 = TRADER_BLACKLIST, //Item unobtaineable
+		/obj/item/device/flashlight/drone		 = TRADER_BLACKLIST, // No drone stuff
+		/obj/item/device/camera_bug				 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/multitool				 = TRADER_BLACKLIST_SUB, // Hacktool, uplink and robo tool ban
+		/obj/item/device/modkit					 = TRADER_BLACKLIST_ALL, // No to modkits
+		/obj/item/device/pin_extractor			 = TRADER_BLACKLIST, // RD's tech
+		/obj/item/device/powersink				 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/slime_scanner			 = TRADER_BLACKLIST, //If it was doing slime stuff, it already had this
+		/obj/item/device/spy_bug				 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/spy_monitor			 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/suit_cooling_unit/improved = TRADER_BLACKLIST, // Not on Aurora
+		/obj/item/device/taperecorder/cciaa		 = TRADER_BLACKLIST, // Admin item
+		/obj/item/device/batterer				 = TRADER_BLACKLIST, // Item too rare
+		/obj/item/device/contract_uplink		 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/uplink                  = TRADER_BLACKLIST_ALL, // Traitor stuff
+		/obj/item/device/announcer				 = TRADER_BLACKLIST, // Rev item
+		/obj/item/device/ninja_uplink			 = TRADER_BLACKLIST, // Ninja item
+		/obj/item/device/onetankbomb			 = TRADER_BLACKLIST, // Not weapons trader
+		/obj/item/device/kinetic_analyzer		 = TRADER_BLACKLIST, // Not KA trader
+		/obj/item/device/camera					 = TRADER_BLACKLIST_SUB, // a lot of ai/drone/cyborg/fluff items
+		/obj/item/device/uv_light				 = TRADER_BLACKLIST, // CSI item
+		/obj/item/device/nanoquikpay			 = TRADER_BLACKLIST, // eftpos exists
+		/obj/item/device/electronic_assembly	 = TRADER_BLACKLIST_ALL, // Not a circuit trader
+		/obj/item/device/integrated_circuit_printer = TRADER_BLACKLIST_ALL, //Not a circuit trader
+		/obj/item/device/integrated_electronics  = TRADER_BLACKLIST_ALL, // Not a circuit trader
+		/obj/item/device/mine_bot_ugprade		 = TRADER_BLACKLIST_ALL, // Not a mining vendor + drone stuff
+		/obj/item/device/mmi					 = TRADER_BLACKLIST_ALL, // Just no to trading cyborg brains
```

Comment:
Why wouldnt a AI trader by mmis?

Reply:
MMI's by law should be considered somewhat sentient - they are able to speak and think, and they are formally sapient beings. I can't imagine there wouldn't be some legislation that wouldn't - however that only applies to subtypes when the MMI is actually filled with a brain.

Maybe allow trading for empty MMI and Empty Radio-Enabled MMI? Posibrains? Your call here, however, lore-wise it would be extremely iffy if the trader accepted "active" mmi's and posibrains.

Reply:
Okay, I just tested it and something worse is happening: There's 3 different MMI's in the code: Normal, Radio enabled and Digital - all three however have the same description - if we allow trade for more than basic mmi, it would lead to frustration on merchant's part as the trader will seemingly randomly decline a perfectly fine MMI. 

I recommend either allowing trading of only basic empty mmi + maybe posibrains, or no mmi trading at all.

![image](https://user-images.githubusercontent.com/6859396/62109423-8eb73080-b2ac-11e9-90f1-cdc6211d89f2.png)

Reply:
> MMI's by law should be considered somewhat sentient
Definitly not the case because the MMI is just the hardware itself.
Blacklist the subtypes and it´s going to be easy enough to recognize and acquire.